src/hotel_reservations/utils.py

```diff
@@ -0,0 +1,124 @@
+import json
+import logging
+import os
+from typing import Any, Optional
+
+import requests
+import yaml
+from databricks.sdk import WorkspaceClient
+from pyspark.ml.evaluation import RegressionEvaluator
+from pyspark.sql import DataFrame
+
+from hotel_reservations.types.project_config_types import ProjectConfig
+
+
+def get_api_token_from_secrets(scope: str, key: str) -> str:
```

Comment:
if you want to do it really properly, look at my suggestion above.

Also, confusing name, it seems like it can return any key from any scope, why do you need to have this function? I mean, w.dbutils.secrets.get(scope=scope, key=key) does the same.
Feels like overengineering for no reason